src/components/contact/ContactCard.vue

```diff
@@ -0,0 +1,125 @@
+<script setup lang="ts">
+import TwitterIcon from "@/assets/icons/twitter.svg?component";
+import FacebookIcon from "@/assets/icons/facebook.svg?component";
+import LinkedinIcon from "@/assets/icons/linkedin.svg?component";
+import InstagramIcon from "@/assets/icons/instagram.svg?component";
+import GithubIcon from "@/assets/icons/github.svg?component";
+import BehanceIcon from "@/assets/icons/behance.svg?component";
+import DribbleIcon from "@/assets/icons/dribble.svg?component";
+import PinIcon from "@/assets/icons/pin.svg?component";
+import EmailIcon from "@/assets/icons/email.svg?component";
+import PhoneIcon from "@/assets/icons/phone.svg?component";
+</script>
+
+<template>
+  <div class="contact-card">
+    <div class="contact-card__map-container">
+      <img
+        src="@/assets/images/map.png"
+        alt="Map of Monterail office location Wroclaw City"
+      />
+    </div>
+    <div class="contact-card__address">
+      <div class="contact-card__contact-icon">
+        <PinIcon />
+      </div>
+      <p>
+        Monterail Sp. z o.o. <br />
+        ul. Oławska 27-29 <br />
+        50-123 Wrocław<br />
+        Poland
+      </p>
+      <div class="contact-card__contact-icon">
+        <EmailIcon />
+      </div>
+      <p>hello@monterail.com</p>
```

Comment:
For reasons of semantics and accessibility, it might be a good idea to put it in a `<a>` tag with a proper protocol https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-href

Reply:
every days is a school day 📖  thank you!